src/Cmfd.cpp

```diff
@@ -2558,7 +2564,7 @@ void Cmfd::generateKNearestStencils() {
         _axial_interpolants.at(fsr_id)[1] = -zc * zc + 26.0/24.0;
         _axial_interpolants.at(fsr_id)[2] = zc * zc/2.0 + zc/2.0 - 1.0/24.0;
 
-        /* Set zero axial prolongation for cells with no fissionalbe material */
+        /* Set zero axial prolongation for cells with no fissionable material */
```

Comment:
num_fissionable_FSRs is nowhere used.

Reply:
True.

Also it looks like the behavior written in the comment is not implemented